lib/chef/knife/profitbricks_ipfailover_remove.rb

```diff
@@ -0,0 +1,60 @@
+require 'chef/knife/profitbricks_base'
+
+class Chef
+  class Knife
+    class ProfitbricksFailoverRemove < Knife
+      include Knife::ProfitbricksBase
+
+      banner 'knife profitbricks ipfailover remove (options)'
+
+      option :datacenter_id,
+             short: '-D DATACENTER_ID',
+             long: '--datacenter-id DATACENTER_ID',
+             description: 'Name of the data center',
+             proc: proc { |datacenter_id| Chef::Config[:knife][:datacenter_id] = datacenter_id }
+
+      option :lan_id,
+             short: '-l LAN_ID',
+             long: '--lan_id LAN_ID',
+             description: 'Lan ID'
+      option :ip,
+             short: '-i',
+             long: '--ip',
+             description: 'IP to be removed from IP failover group'
+
+      option :nic_id,
+             short: '-n',
+             long: '--nic_id',
+             description: 'NIC to be removed from IP failover group'
+
+      def run
+        $stdout.sync = true
+        validate_required_params(%i[datacenter_id lan_id ip nic_id], Chef::Config[:knife])
+
+        connection
+
+        lan = ProfitBricks::LAN.get(Chef::Config[:knife][:datacenter_id], Chef::Config[:knife][:lan_id])
+
+        ipfailover = lan.properties['ipFailover']
+
+        ipfailover.each_with_index do |value, index|
+          if value['nicUuid'] == Chef::Config[:knife][:nic_id] && value['ip'] == Chef::Config[:knife][:ip]
```

Comment:
Can we use nic_id instead of Uuid here? Just to keep the `id` reference consistent throughout the code.

Reply:
the object ipFailover in ruby SDK is like that we can not pass anything else.